### DISCUSSION TITLE
After update from 1.1.9 to 1.2.2 I can´t connect

### DISCUSSION THREAD

**Discussion by yardstudio** [rustdesk/rustdesk#5912] [General]
### Bug Description

I am using version 1.1.9 without problems before (only bug with diacritics), then I updated to 1.2.2 and after that, I can´t connect. 
Rust Desk repeatedly show Green "Ready. For faster..." then switching to Red "Searching server" and "Not ready.." - look at video
https://screenpal.com/watch/c06fIrV5U04
![2023-10-04_14-56-51](https://github.com/rustdesk/rustdesk/assets/25852179/6c1e3835-28be-4eb7-8353-887467458a08)
![2023-10-04_14-55-27](https://github.com/rustdesk/rustdesk/assets/25852179/55b80849-0f00-4d78-ada9-fc45d61b9173)
![2023-10-04_14-56-59](https://github.com/rustdesk/rustdesk/assets/25852179/3717f778-5f38-44a3-8883-4a2b532506f9)

### How to Reproduce

Upgrade old version 1.1.9 to 1.2.2 ( not uninstalled old version, only run new installer and reinstall accross)

### Expected Behavior

Expected stable connect, not possible connect to another comp on the same LAN (connecting accross Internet)

### Operating system(s) on local side and remote side

Windows 10 Pro

### RustDesk Version(s) on local side and remote side

1.2.2 - 1.2.2

### Screenshots

![2023-10-04_14-29-18](https://github.com/rustdesk/rustdesk/assets/25852179/14c96bf3-d06c-46cc-ba40-aaec342e709a)
![2023-10-04_14-59-06](https://github.com/rustdesk/rustdesk/assets/25852179/35f39092-d117-4041-8dcb-08fac772bd9c)
![2023-10-04_15-02-02](https://github.com/rustdesk/rustdesk/assets/25852179/0ad0de3b-5d02-4afb-bc87-e907684ff828)
![2023-10-04_14-56-51](https://github.com/rustdesk/rustdesk/assets/25852179/610f695d-44c5-4e2f-89b0-8fab47f7de65)

**Comment by godzfire** [comment]
Same issue here, however I was on 1.2.2 for awhile so I don't think it's that and just a connectivity outage with Rustdesk. It was working yesterday.

**Comment by rjcomm** [comment]
Same here.
Most of the computers where Rustdesk is installed I am now unable to connect using the Rustdesk servers for access. Seemed to be more of a problem lately after updating from V1.9x to 1.2.x versions. I believe it worked with version 1.2.2 for a short time, but as of the last week no luck. Having to use alternative remote access software.
I very much like RustyDesk however and never had a problem early on with V1.9

**Comment by rustdesk** [comment]
Our public server is over loaded lately, self-host please.

**Reply by godzfire** [reply]
I'm just a single user, I don't have the ability or knowledge to do this. As a result, it leaves me unable to use Rustdesk and likely have to switch to a different remoting software, which is the reason I originally came over to Ruskdesk from Anydesk, because of all the good I had heard about it.

**Comment by godzfire** [comment]
I'm just a single user, I don't have the ability or knowledge to do this. As a result, it leaves me unable to use Rustdesk and likely have to switch to a different remoting software, which is the reason I originally came over to Ruskdesk from Anydesk, because of all the good I had heard about it.

**Comment by grahamIT** [comment]
For anyone wondering, this has nothing to do with the Rustdesk version. It is caused by the main rustdesk server being down due to high demand. You can learn how to self host here: https://rustdesk.com/docs/en/self-host/rustdesk-server-oss/

I personally just set it up on a raspberry pi that was hosting a vpn. I  used docker compose in order to make it easy.

**Reply by godzfire** [reply]
Check my reply above in regards to that. For the average user, they don't have the knowledge on how to do such a thing. It's the reason they sought a remove program like this in the first place. If that's the case, people should just go back to AnyDesk or Teamviewer.

**Reply by grahamIT** [reply]
Its up to the admins to deal with that. That's why I went ahead and setup my own server on prem for everyone to use.

**Reply by yardstudio** [reply]
Why old version have not this issue? It use the same server, or not?

**Comment by Leprechaun777** [comment]
I'm still having the same problem with 1.2.3 on a Raspberry Pi. 
I'm assuming the servers are ok now.

**Comment by Stefeno** [comment]
I've had the same problem for 1 year now! I'm forced to go back to anydesk unfortunately... I'm sorry
